### ISSUE TITLE
V1 - Control Objective - is "security" a "security principle" in itself?

### ISSUE THREAD

**Issue by securitybits** [OWASP/ASVS#549]
Last paragraph:
"In this chapter, the ASVS covers off the primary aspects of any sound security architecture: **security**, availability, confidentiality, processing integrity, and privacy. Each of these security principles must be..."

Security is usually defined as a combination of various properties we want to maintains, such as confidentiality, integrity, availability, authenticity, privacy - but listing security as part of the primary aspect of sound security architecture appears self-referencing rather than defining to me. What is intended with security in this listing, and how does it differ from the other aspects?

**Comment by tghosth** [comment]
I addressed the basic point that the word security is redundant in https://github.com/OWASP/ASVS/commit/b9e13222b0975db446fb2d83932800cb9eea245f however this raises a wider point for whether the sentance "[In this chapter, the ASVS covers off the primary aspects of any sound security architecture: availability, confidentiality, processing integrity, and privacy.](https://github.com/OWASP/ASVS/blob/master/4.0/en/0x10-V1-Architecture.md#control-objective)" really makes sense after the architecture restructure.
Thoughts @vanderaj @jmanico @danielcuthbert ?

**Comment by vanderaj** [comment]
I'm pretty happy with Josh's re-wording here. Considering the amount of auditing and logging controls, I added non-repudiation. It's great to finally have a standard that can clearly set out to meet that need.

**Comment by vanderaj** [comment]
For a more philosophical view on the question you posed, I posit that we do cover off all of those elements throughout the ASVS. We build on them bit by bit throughout, with security architecture being the foundstone of the rest. I think in 4.1, we should probably try to workshop V1 with some enterprise security architects as I think there are areas that I used to know back in the day when i was doing literally billion dollar+ corporate banking platforms, that I think I've overlooked or forgotten now.
